.eslintrc.js

```diff
@@ -26,4 +26,12 @@ module.exports = {
         sourceType: "module",
         ecmaVersion: 2022,
     },
+    overrides: [
+        {
+            files: "utils/*",
+            rules: {
+                "no-console": "off",
+            },
```

Comment:
In our utils, I think it's useful to be able to log output to the console for local dev but also so when the action has issues we have more info.